frontend/messages/fr.json

```
@@ -953,20 +953,152 @@
+	"privacy_cloud_provider": "Fournisseur de Services Cloud",
+	"privacy_it_provider": "Fournisseur de Services Informatiques",
+	"privacy_marketing_agency": "Agence de Marketing",
+	"privacy_payment_processor": "Processeur de Paiement",
+	"privacy_analytics_provider": "Fournisseur d'Analyse",
+	"privacy_business_partner": "Partenaire Commercial",
+	"privacy_distributor": "Distributeur",
+	"privacy_reseller": "Revendeur",
+	"privacy_supplier": "Fournisseur",
+	"privacy_contractor": "Contractant",
+	"privacy_legal_advisor": "Conseiller Juridique",
+	"privacy_accountant": "Comptable",
```

Biome (1.9.4)

[error] The key privacy_employee was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)